site/src/components/accordion.md

```diff
@@ -0,0 +1,165 @@
+	</div>
+</details>
+<details class="accordion">
+	<summary>
+		<div class="accordion-header">Accordion header 2</div>
+	</summary>
+	<div class="accordion-content">
+		<p>Accordion panel 2</p>
+	</div>
+</details>
+<details class="accordion">
+	<summary>
+		<div class="accordion-header">Accordion header 3</div>
+	</summary>
+	<div class="accordion-content">
+		<p>Accordion panel 3</p>
+	</div>
+</details>
+```
+
+### Name
+
+Applying the `name` attribute to all accordion items with the same value will allow opening only one of the items at a time.
+
+```html
+<details class="accordion" name="accordion-group-1">
+	<summary>
+		<div class="accordion-header">Accordion header 1</div>
+	</summary>
+	<div class="accordion-content">
+		<p>Accordion panel 1</p>
+	</div>
+</details>
+<details class="accordion" name="accordion-group-1">
+	<summary>
+		<div class="accordion-header">Accordion header 2</div>
+	</summary>
+	<div class="accordion-content">
+		<p>Accordion panel 2</p>
+	</div>
+</details>
+<details class="accordion" name="accordion-group-1">
+	<summary>
+		<div class="accordion-header">Accordion header 3</div>
+	</summary>
+	<div class="accordion-content">
+		<p>Accordion panel 3</p>
+	</div>
+</details>
+```
+
+## Icon location
+
+By default, the expansion indicator icon is located before the accordion item's header. Applying the `.accordion-icon-end` class to the item will change the icon's position to the end of the header.
+
+```html
+<details class="accordion">
+	<summary>
+		<div class="accordion-header">Accordion icon's default position</div>
+	</summary>
+	<div class="accordion-content">
+		<p>Accordion panel 1</p>
+	</div>
+</details>
+<details class="accordion accordion-icon-end border-top">
+	<summary>
+		<div class="accordion-header">Accordion icon's end position</div>
+	</summary>
+	<div class="accordion-content">
+		<p>Accordion panel 2</p>
+	</div>
+</details>
+```
+
+## Sizes
+
+`accordion-sm`/`accordion-lg` modifier classes make the text size and padding a little bigger or smaller than the default.
+
+```html
+<details class="accordion accordion-sm">
+	<summary>
+		<div class="accordion-header">Small accordion header</div>
+	</summary>
+	<div class="accordion-content">
+		<p>Accordion panel content</p>
+	</div>
+</details>
+<details class="accordion">
+	<summary>
+		<div class="accordion-header">Default accordion header</div>
+	</summary>
+	<div class="accordion-content">
+		<p>Accordion panel content</p>
+	</div>
+</details>
+<details class="accordion accordion-lg">
+	<summary>
+		<div class="accordion-header">Large accordion header</div>
+	</summary>
+	<div class="accordion-content">
+		<p>Accordion panel content</p>
+	</div>
+</details>
+```
+
+## Patterns
+
+This page discusses the component and subcomponents that make up `.accordion`, but it does not prescribe particular patterns. For examples of specific accordion builds, look to the [accordion patterns page](../patterns/accordion.md) for guidance.
```

Comment:
This is a great disclaimer that I nearly missed because it's at the bottom. Could we find a way to incorporate this in the introductory section?

Reply:
Moved it up to the intro section.